app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -136,7 +136,7 @@ def track_email_click
     end
 
     def redirect_to_completed_dashboard_if_not_accepted
-      redirect_to candidate_interface_application_complete_path if !any_accepted_offer? || FeatureFlag.inactive?(:new_references_flow)
+      redirect_to candidate_interface_application_complete_path if (!any_accepted_offer? && !current_application.recruited?) || FeatureFlag.inactive?(:new_references_flow)
```

Comment:
extract `(!any_accepted_offer? && !current_application.recruited?)` to suitably-named little method?